spec/openstates/api_spec.rb

```diff
@@ -45,6 +45,14 @@
         OpenStates.bills(options)
       end
     end
+
+    context "with the id passed" do
+      it "should call .get with 'bills/10' and an options hash" do
+        options = {:id => 10}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.